### DISCUSSION TITLE
allow only one id column?

### DISCUSSION THREAD

**Discussion by Yangdongjin** [HadesArchitect/GrafanaCassandraDatasource#140] [Q&A] [answered]
my primary key is composite key.
exam) 
table enegy(
    name text,
    age int,
    sex text,
    time timestamp,
    gas_use float
    primary key((name, age), time)
)
how to use composite primary key(name, age) in grafana?

**Comment by HadesArchitect** [comment, marked as answer]
Hey, you will need to use query editor instead of query configurator as explained [here](https://github.com/HadesArchitect/GrafanaCassandraDatasource/wiki/User-Guide#query-editor)
